ocfweb/docs/docs/services/hpc.md

```diff
@@ -0,0 +1,183 @@
+[[!meta title="High Performance Computing"]]
```

Comment:
Another thing I just noticed: this should be `High performance computing` in accordance with our title convention